3.3/examples/django/README.md

```diff
@@ -0,0 +1,91 @@
+Django application example
+==========================
+
+This is a basic [Django](https://www.djangoproject.com/) project and some helper scripts to demonstrate how to deploy Python web applications with [OpenShift 3](http://www.openshift.org).
+
+We will be using Django 1.8 and the [PostgreSQL](http://www.postgresql.org/) database.
+
+OpenShift supports multiple deployment strategies. This example uses the Source-To-Image strategy, and the [Python 3.3 STI image](https://github.com/openshift/sti-python/tree/master/3.3) for producing application builds.
+
+**WARNING**: this is a _proof of concept_ example and, therefore, the code available here is not meant to be used in production.
+
+
+Application Build, Deploy, and Update Flow
+------------------------------------------
+
+Start by cloning this repository and changing your current directory appropriately:
+
+```bash
+git clone https://github.com/openshift/sti-python.git
+cd sti-python/3.3/examples/django
+```
+
+Follow the instructions in https://github.com/openshift/origin/tree/master/examples/sample-app to build the `openshift` all-in-one binary and start your OpenShift 3 server and docker registry. You won't need to clone that repository, and you should use `sti-python/3.3/examples/django` as your working directory.
+
+Before proceeding, make sure you have the OpenShift server with a docker registry, a project and a user.
+
+
+### Building your app ###
+
+In the future, creating a new app in OpenShift from a git repo should automatically detect what kind of application it is, and create some entities to support it.
+
+For now these entities are declared explicitly in the `application-template-stibuild.json` file, which is a template we use to create the entities.
+You may want to inspect this file and tweak some configuration. Then, go on and create the entities:
+
+```bash
+osc process -f application-template-stibuild.json | osc create -f -
+```
+
+A build of your app should have been triggered automatically. Wait until the build completes with this command:
+
+```bash
+osc get builds --watch
+```
+
+The STI build process essentially copies your source files and install dependencies listed in `requirements.txt`, finally producing a new Docker image.
+
+
+### Application deployment ###
+
+An application deployment is triggered right after a build because of the ImageChange trigger.
+
+Your database pod was deployed because of the ConfigChange trigger.
+
+You should now the able to see your Django application running. To find out the IP and port of your service, run this:
+
+```bash
+osc describe service frontend
+```
+
+
+### One-off commands ###
+
+The Django app requires the execution of some initial one-off commands in order finish setting it up.
```

Comment:
re: missing "to", added.
re: initial steps, this is not a Django tutorial. And those steps will change, should change. `./manage.py collectstatic` should be part of the build process, and `./manage.py migrate` part of the deployment process.
These are in the scope of the quickstart I'm working on: https://github.com/rhcarvalho/openshift-django-quickstart